src/js/letters/containers/Main.jsx

```diff
@@ -10,9 +12,19 @@ class Main extends React.Component {
   }
 
   render() {
+    let appContent;
+
+    if (this.props.lettersAvailable) {
+      appContent = this.props.children;
+    } else if (this.props.lettersAvailable === undefined) {
```

Comment:
The downside to doing this is that since `lettersAvailable` is a boolean, it's surprising for it to have three different states instead of two. It's also easy to forget and add a check that catches both `false` and `undefined`, since they're both false-y. 

It might not be an issue if these two apps are pretty small and simple, but if you end up needing to add different failure states, you might end up wanting a loading or status property to make the logic clearer.